charts/lnd/templates/statefulset.yaml

```diff
@@ -226,11 +250,31 @@ spec:
         - name: configmap
           configMap:
             name: {{ include "lnd.fullname" . }}
-        {{- if ne .Values.global.network "regtest" }}
+        {{- if or (ne .Values.global.network "regtest") .Values.lndWatchOnly.enabled }}
         - name: lnd-password
           secret:
             secretName: {{ printf "%s-pass" (include "lnd.fullname" .) }}
         {{- end }}
+        {{- if .Values.lndWatchOnly.enabled }}
+        - name: signer-tls
+          secret:
+            secretName: signer1-credentials
```

Comment:
Why is there a "1" suffix for "signer" in the secret name?

Reply:
Similar to the full featured LND nodes the signer nodes can be two as well paired with the watch-only lnd-s.
If we run two nodes in the cluster the same distinction is needed. Will be more apparent on staging.

Reply:
But this code is hard coding the lnd sts to only work with secrets whose name is `signer1-credentials`. If we wanted to use lnd2 too with its own remote signer, it wouldn't work, because lnd2's sts would reference `signer1-credentials` instead of `signer2-credentials`

Reply:
fixed in [111ff6a](https://github.com/GaloyMoney/charts/pull/1581/commits/111ff6a6bbbcc5ee2af7d0b162bf3e5fb6c5ba67)